components/builder-header/__examples__/base-with-toolbar.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import IconSettings from '../../icon-settings';
+import Button from '../../button';
+import ButtonGroup from '../../button-group';
+import Dropdown from '../../menu-dropdown';
+import BuilderHeader from '../../builder-header';
+import BuilderHeaderNav from '../../builder-header/nav';
+import BuilderHeaderNavLink from '../../builder-header/nav-link';
+import BuilderHeaderToolbar from '../../builder-header/toolbar';
+
+const Example = (props) => (
+	<IconSettings iconPath="/assets/icons">
+		<BuilderHeader
+			assistiveText={{
+				backIcon: 'Back',
+				helpIcon: 'Help',
+				icon: 'Builder',
+			}}
+			labels={{
+				back: 'Back',
+				help: 'Help',
+				pageType: 'Page Type',
+				title: 'App Name',
+			}}
+			style={{ position: 'relative' }}
```

Comment:
This style override was needed because the header has `position: fixed`, so it was showing up at the top of the doc site.